game/client_types/logic.js

```diff
@@ -87,7 +87,7 @@ module.exports = function(treatmentName, settings, stager, setup, gameRoom) {
                     txt = cacheToSave.join("\n") + "\n";
                     cacheToSave = [];
                     timeOutSave = null;
-                    fs.appendFile(gameRoom.dataDir + 'codes.csv', txt,
+                    fs.appendFile(gameRoom.dataDir + '.codes.csv', txt,
```

Comment:
This is not correct, still does not save inside the room folder. Use the path module to join paths.

```
path.join(gameRoom.dataDir, codes.csv)
```